jinahub/loaders/AudioLoader/config.yml

```diff
@@ -0,0 +1,4 @@
+jtype: AudioLoader
+metas:
+  py_modules:
+    - executor/audio_loader.py
```

Comment:
Since it is a single-file executor, there is no need to have the folder structure. Check out https://docs.jina.ai/fundamentals/executor/repository-structure/#single-python-file